samples/search_vm_from_string.py

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env python
+# VMware vSphere Python SDK
+# Copyright (c) 2008-2013 VMware, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
```

Comment:
Hi @rjrpaz , Thanks for contributing.
I believe instead of writing complete new sample, You can add '-f ' command line arg to Existing [getAllVMs](https://github.com/vmware/pyvmomi-community-samples/blob/master/samples/getallvms.py) sample. 

Thanks

Reply:
Yes, I agree. Done as suggested.